frontend/public/components/overview/OverviewListPage.tsx

```diff
@@ -1,13 +1,10 @@
 import * as React from 'react';
-import { useParams } from 'react-router-dom';
-import DataModelProvider from '@console/dev-console/src/components/topology/data-transforms/DataModelProvider';
-import { TopologyDataRenderer } from '@console/dev-console/src/components/topology/TopologyDataRenderer';
+import TopologyPage from '@console/dev-console/src/components/topology/TopologyPage';
```

Comment:
This import looks bad. We shouldn't import the dev-console package in public.

Reply:
This is known and already existing. I believe the plan is move Topology out of dev-console and into its own mono repo.

Reply:
If we are to share topology with admin console, we need to move it to a new package as a standalone tool.